wagtail_localize/templates/wagtail_localize/admin/submit_translation.html

```diff
@@ -0,0 +1,47 @@
+
+{% extends "wagtailadmin/base.html" %}
+{% load i18n wagtailadmin_tags %}
+{% block titletag %}{{ view.get_title }} {{ view.get_subtitle }}{% endblock %}
+
+{% block content %}
+    {% include "wagtailadmin/shared/header.html" with title=view.get_title subtitle=view.get_subtitle icon="doc-empty-inverse" %}
+
+    <div class="nice-padding">
+        <form method="POST">
+            {% csrf_token %}
+
+            {% if next_url %}
+                <input type="hidden" name="next" value="{{ next_url }}">
+            {% endif %}
+
+            {% for field in form.hidden_fields %}{{ field }}{% endfor %}
+
+            <ul class="fields">
+                {% block visible_fields %}
+                    {% for field in form.visible_fields %}
+                        {% include "wagtailadmin/shared/field_as_li.html" %}
+                    {% endfor %}
+                {% endblock %}
+                <li><input type="submit" value="{% trans 'Submit' %}" class="button" />{% if back_url %} <a href="{{ back_url }}" class="button button-secondary">{% trans "Go back" %}</a>{% endif %}</li>
+            </ul>
+        </form>
+    </div>
+{% endblock %}
+
+
+{% block extra_js %}
+    {{ block.super }}
+
+    <script type="text/javascript">
+        document.addEventListener('DOMContentLoaded', function() {
+            var selectAll = document.querySelector('[name="select_all"]');
+            var locales = document.querySelectorAll('[name="locales"]');
+
+            selectAll.addEventListener('change', function() {
+                locales.forEach(function(locale) {
```

Comment:
We'll run into IE11 issues with `.forEach()` https://caniuse.com/#search=forEach

Reply:
Not thought much about browser support. Since this is a new plugin, do you think we need to support IE11?

Reply:
Implemented a work around for this now!